client/src/components/dashboard/DashboardHeader.test.js

```diff
@@ -0,0 +1,24 @@
+  
```

Comment:
newline

Reply:
eslint style requirement

Reply:
you don't have this in your other test files though

Reply:
sorry, I didn't see that u were talking about the tests. I will remove all spaces for tests. but keep for component